applications/plugins/LMConstraint/src/LMConstraint/BarycentricDistanceLMConstraintContact_DistanceGrid.cpp

```diff
@@ -48,7 +47,6 @@ Creator<Contact::Factory, BarycentricDistanceLMConstraintContact<FFDDistanceGrid
 Creator<Contact::Factory, BarycentricDistanceLMConstraintContact<FFDDistanceGridCollisionModel, SphereCollisionModel<sofa::defaulttype::Vec3Types>> > FFDDistanceGridSphereDistanceLMConstraintContactClass("distanceLMConstraint",true);
 Creator<Contact::Factory, BarycentricDistanceLMConstraintContact<FFDDistanceGridCollisionModel, TriangleCollisionModel<sofa::defaulttype::Vec3Types>> > FFDDistanceGridTriangleDistanceLMConstraintContactClass("distanceLMConstraint",true);
 
-
 } // namespace collision
```

Comment:
Last thing, it would be good to keep consistency between files about the namespaces format (nested or not)

Reply:
not really but I was mostly saying that because some other files have the new format (e.g ContactDescription.h) so it is a bit weird to have both.